content/posts/october-2020

```diff
@@ -0,0 +1,14 @@
+---
+title: "October 2020"
+date: 2020-10-15
+draft: false
+---
+
+## Week 41 (October 5th - October 9th)
+
+- [franta] improved the way we handle jobs. Triggering of them is more reliable -
+  they are being triggered in correct situations ([packit-service#837](https://github.com/packit/packit-service/pull/837))
+
+- We've enhanced the stability of packit-service by adding multiple improvements in our testing procedures.
```

Comment:
I don't think users care about our testing. Feel free to remove this :-)

Reply:
They care about stability, but sure :)